subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/TestProjectGeneratorConfiguration.groovy

```diff
@@ -134,6 +136,7 @@ class TestProjectGeneratorConfigurationBuilder {
         config.testRunnerMemory = '256m'
         config.parallel = this.subProjects > 0
         config.systemProperties = this.systemProperties
+        config.featurePreviews = this.featurePreviews
```

Comment:
I am surprised we don't already test other feature previews in our performance tests...

Reply:
Which ones do you refer to? Maybe you can `@` the people here...

Reply:
I asked on Slack.

Reply:
So looks like we did enable the features in an ad-hoc fasion.